### PR TITLE
TFTRT: Prevent segments with no inputs

### PR DESCRIPTION
**Pull request by trevor-m**
If a segment is created with no inputs, the TRTEngineOp created by TFTRT currently crashes during runtime. This PR simply prevents the engine for that segment from building so that the native TF fallback will be executed instead of crashing.

This issue came up for me recently trying to convert ConvS2S which uses weight normalization. I've created an issue for the potential bug in constant folding: https://github.com/tensorflow/tensorflow/issues/24083.

This may become redundant when the node validation is working for all nodes - that would disqualify these nodes from even being considered for a segment because their inputs are weights not tensors. Nevertheless it is a good check to have.

### CODE REVIEW

**Comment by pooyadavoodi** [comment]
Looks good to me.

@azaks2 @smit-hinsu 
Could you please review.

**Comment by smit-hinsu** [comment]
Looks good to me and will approve once https://github.com/tensorflow/tensorflow/pull/24183 is merged to update the CODEOWNERS file.

File: tensorflow/contrib/tensorrt/convert/convert_graph.cc
Comment (smit-hinsu) [inline tensorflow/contrib/tensorrt/convert/convert_graph.cc]:
```suggestion
  if (inputs.empty()) {
```

**Review by smit-hinsu — CHANGES_REQUESTED** [review]
Minor suggestion.

**Review by smit-hinsu — DISMISSED** [review]

**Comment by smit-hinsu** [comment]
Trevor, the commit to use `empty` method has your personal email address as author and that email has not signed the CLA.

**Comment by trevor-m** [comment]
> Trevor, the commit to use `empty` method has your personal email address as author and that email has not signed the CLA.

Oh I see, thanks for pointing that out.

**Review by smit-hinsu — APPROVED** [review]

**Pull request merged**
